### PR TITLE
fix(express): Add package.json for express

### PR DESCRIPTION
**Pull request by quinnturner**
Adds a versioned dependency for `@types/express` for `@types/express-serve-static-core` `^4.17.18`.
This fixes a breaking typing issue for `@types/4.17.10` where it strictly requires `@types/express-serve-static-core` `^4.17.18`.
Otherwise, all properties on `Request` are not defined.

Please review the existing discussion on this thread: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50390

~We might be required to modify: https://github.com/microsoft/DefinitelyTyped-tools/blob/master/packages/definitions-parser/allowedPackageJsonDependencies.txt~ Modification merged.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

File: types/express/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "dependencies": {
+    "@types/body-parser": "*",
+    "@types/express-serve-static-core": "^4.17.18",
```
Comment (andrewbranch) [inline types/express/package.json:5]: Remove the others and add this one here: https://github.com/microsoft/DefinitelyTyped-tools/blob/master/packages/definitions-parser/allowedPackageJsonDependencies.txt

Comment (quinnturner) [reply types/express/package.json:5]: I was thinking that as well, glad I got the second opinion. Thanks!

**Comment by quinnturner** [comment]
Submitted a request on `allowedPackageJsonDependencies`

https://github.com/microsoft/DefinitelyTyped-tools/pull/193

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## express ([<kbd>unpkg</kbd>](https://unpkg.com/browse/express@latest/))
was missing the following properties:
1. Route
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against bb385e1224e07264d5aef8e63a8c077b79219298
</p>

**Comment by andrewbranch** [comment]
@AndrewLeedham, can you sign off?

**Review by AndrewLeedham — APPROVED** [review]
Yes this looks good to me

**Review by andrewbranch — APPROVED** [review]

**Comment by quinnturner** [comment]
Tested, here's my current `yarn.lock`:

```
"@types/express@*", "@types/express@^4.17.11":
  version "4.17.11"
  dependencies:
    "@types/body-parser" "*"
    "@types/express-serve-static-core" "^4.17.18"
    "@types/qs" "*"
    "@types/serve-static" "*"
```

LGTM ✅

**Pull request merged**